migrate-from-csv-files-to-tidb.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Migrate Data from CSV Files to TiDB
 summary: Learn how to migrate data from CSV files to TiDB.
+aliases: ['/docs/dev/tidb-lightning/migrate-from-csv-using-tidb-lightning/','/docs/dev/reference/tools/tidb-lightning/csv/','/tidb/dev/tidb-lightning/migrate-from-csv-using-tidb-lightning']
```

Comment:
```suggestion
aliases: ['/docs/dev/tidb-lightning/migrate-from-csv-using-tidb-lightning/','/docs/dev/reference/tools/tidb-lightning/csv/','/tidb/dev/migrate-from-csv-using-tidb-lightning']
```

Reply:
https://docs.pingcap.com/tidb/dev/migrate-from-csv-using-tidb-lightning will be redirected.